api/src/main/java/net/kyori/adventure/audience/Audience.java

```diff
@@ -429,13 +429,26 @@ default void hideBossBar(final @NonNull BossBar bar) {
   }
 
   /**
-   * Plays a sound.
+   * Plays a sound at the location of the receiver.
    *
    * @param sound a sound
    * @see Sound
    * @since 4.0.0
    */
   default void playSound(final @NonNull Sound sound) {
+    this.playSound(sound, false);
+  }
+
+  /**
+   * Plays a sound.
+   * <p>Custom sounds cannot be made to follow the receiver and they will fall back to playing at the location of the receiver.</p>
+   *
+   * @param sound a sound
+   * @param followReceiver if the sound should follow the receiver
+   * @see Sound
+   * @since 4.8.0
+   */
+  default void playSound(final @NonNull Sound sound, final boolean followReceiver) {
```

Comment:
It makes me a bit uncomfortable that things that previously overrode `playSound` will now give no behavior when `playSound(sound, false)` is called, but without some crazy shenanigans it's hard to fix that.

Reply:
Wouldn't be that hard to fix, we can make this method default to call the old `playSound` method and leave that one as is. I believe this is how it's done elsewhere in the codebase - I think in the boss bar listener percent/progress thing?